client/settings/payment-methods-selector/style.scss

```
@@ -0,0 +1,39 @@
+.woocommerce-payments__payment-method-selector__separator {
+	margin: 24px -24px;
+}
+
+.woocommerce-payments__payment-method-selector__list-item {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	width: 100%;
+
+	& > * {
+		margin: 0 3px;
+	}
+
+	.woocommerce-payments__payment-method-selector__list-item-fees {
```

You can use the `&` to prevent repetition
```suggestion
	&-fees {
```

But since `woocommerce-payments__payment-method-selector__list-item-fees` is a child of `woocommerce-payments__payment-method-selector__list-item`, by convention should it be called `woocommerce-payments__payment-method-selector__list-item__fees`?

---

:+1: